test/src/test/java/mil/nga/giat/geowave/test/query/AttributesSubsetQueryIT.java

```diff
@@ -0,0 +1,372 @@
+package mil.nga.giat.geowave.test.query;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.List;
+import java.util.UUID;
+
+import mil.nga.giat.geowave.adapter.vector.FeatureDataAdapter;
+import mil.nga.giat.geowave.core.geotime.GeometryUtils;
+import mil.nga.giat.geowave.core.geotime.IndexType;
+import mil.nga.giat.geowave.core.geotime.store.query.SpatialQuery;
+import mil.nga.giat.geowave.core.store.CloseableIterator;
+import mil.nga.giat.geowave.core.store.DataStore;
+import mil.nga.giat.geowave.core.store.index.Index;
+import mil.nga.giat.geowave.datastore.accumulo.AccumuloDataStore;
+import mil.nga.giat.geowave.datastore.accumulo.BasicAccumuloOperations;
+
+import org.apache.accumulo.core.client.AccumuloException;
+import org.apache.accumulo.core.client.AccumuloSecurityException;
+import org.apache.accumulo.minicluster.MiniAccumuloCluster;
+import org.apache.accumulo.minicluster.MiniAccumuloConfig;
+import org.apache.commons.io.FileUtils;
+import org.apache.log4j.Logger;
+import org.geotools.data.DataUtilities;
+import org.geotools.feature.SchemaException;
+import org.geotools.feature.simple.SimpleFeatureBuilder;
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.opengis.feature.simple.SimpleFeature;
+import org.opengis.feature.simple.SimpleFeatureType;
+
+import com.google.common.io.Files;
+import com.vividsolutions.jts.geom.Coordinate;
+import com.vividsolutions.jts.geom.Envelope;
+
+public class AttributesSubsetQueryIT
```

Comment:
@rfecher 
Thoughts on going this way vs. inheriting from one of the GeoWaveServices classes?

@dcy2003 
My initial inclination would be to move to inherit from one of the existing service environments (maybe GeoWaveTestEnvironment ) - that's the pattern the other integration tests use.  The though behind it would be to amortize the cost of setting up/configuration a miniaccumulocluster.
Caveated  - this is just a quick off the cuff comment, so there may be a good reason not to do this - but absent that it's probably the pattern we would stick to.   Basically you should just mostly be able to get rid of declaring your own accumulo member, tempdir, user, pass, and use instance variables.   otherwise the rest of the code should be about the same.   ( see: https://github.com/ngageoint/geowave/blob/master/test/src/test/java/mil/nga/giat/geowave/test/GeoWaveBasicIT.java )

Also you might need to register this test here:
https://github.com/ngageoint/geowave/blob/master/test/src/test/java/mil/nga/giat/geowave/test/GeoWaveITSuite.java